### PR TITLE
Updated the commands of installing multi-cloud server

### PR DESCRIPTION
**Pull request by shelar1423**
issue: #195 

- Updated the commands of installing multi-cloud server After mkdir command a cd command  added where we will clone the multi-cloud repo.
 
#sodacode2022
#sodalow 
regards 
your contributor
Digvijay shelar 
@shelar1423

### CODE REVIEW

**Comment by shelar1423** [comment]
@skdwriting  kindly label sodalow and sodacode2022

File: content/guides/developer-guides/multi-cloud/_index.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ docker-compose version 1.22.0, build 1719ceb
 ### Install and start SODA Multi-cloud Server
 Download the source code:
 ```bash
-mkdir -p /root/gopath/src/github.com/sodafoundation/
+mkdir cd -p /root/gopath/src/github.com/sodafoundation/
```
Comment (anvithks) [inline content/guides/developer-guides/multi-cloud/_index.md:64]: What does this command do? We already use mkdir with the -p option to create the directory structure.
The correct way would be to provide the path in the clone command as :  
`git clone https://github.com/sodafoundation/multi-cloud.git /root/gopath/src/github.com/sodafoundation/`

or  

Change the order of the commands. Navigate to the `/root/gopath/src/github.com/sodafoundation/` folder first and then clone.

File: content/guides/developer-guides/multi-cloud/_index.md [hunk 2]
```diff
@@ -108,4 +108,4 @@ ad7a85f51343        wurstmeister/zookeeper                 "/bin/sh -c '/usr/sb
 0722d5174b8f        pingcap/tidb:v2.1.16                   "/tidb-server --stor…"   42 seconds ago      Up 41 seconds       0.0.0.0:4000->4000/tcp, 0.0.0.0:10080->10080/tcp     tidb
 e03e03d88dcb        sodafoundation/multi-cloud-block       "/block"                 42 seconds ago      Up 40 seconds                                                            multi-cloud_block_1_7adafa5c5b8d
 edc2036f5e1f        redis                                  "docker-entrypoint.s…"   42 seconds ago      Up 41 seconds       0.0.0.0:6379->6379/tcp                               redis
-```
\ No newline at end of file
+```
```
Comment (anvithks) [inline content/guides/developer-guides/multi-cloud/_index.md:111]: What is changed here?

**Review by anvithks — CHANGES_REQUESTED** [review]
Please check the review comments. Either of the solutions will work.

**Comment by shelar1423** [comment]
> Please check the review comments. Either of the solutions will work.

Made changes according to your comments have a look

**Comment by anvithks** [comment]
@shelar1423 this is a duplicate of #197

**Pull request closed without merging**